### PR TITLE
tableview-ocrvs-2543: Part 2- Listview Component

### PR DESCRIPTION
**Pull request by soumita-dsi**

### CODE REVIEW

**Comment by jpye-finch** [comment]
Looks good @soumita-dsi 

Are we missing varients:
- With Avatar
- With multiple action buttons eg. Primary and CircleButton

File: packages/client/src/views/Home/RecordAudit.tsx [hunk 1]
```diff
@@ -240,6 +240,48 @@ const getLocation = (
   return ''
 }
 
+/*const getSavedApplications = (props: IFullProps): IApplicationData => {
+  const savedApplications = props.savedApplications
+  const applicationId = props.match.params.applicationId
+
+  const applications = savedApplications
```
Comment (atiqzaman-dsi) [inline packages/client/src/views/Home/RecordAudit.tsx:247]: Do you want to delete this commented code?

**Review by atiqzaman-dsi — CHANGES_REQUESTED** [review]

**Comment by soumita-dsi** [comment]
> Looks good @soumita-dsi
> 
> Are we missing variants for ListRow?
> 
> * With Avatar
> * With multiple action buttons eg. Primary and CircleButton
> * With Tag (status eg Active) and CircleButton
>   https://www.figma.com/file/RYc1inuUINR1PvEZq4dVjyWF/components?node-id=4429%3A3224
> 
> Not clear on the need for HideOnDesktop and HideOnMobile...?
> 
> * what are MenuContainer and StatusContainer used for?

Thanks @jpye-finch. I have tried to cover all the variants in ListRow component.
  -> Avatar can be passed in the nameWithAvatar props of this component.
 -> Yes, primary and circle button can be passed in action and actionsMenu props

**Comment by soumita-dsi** [comment]
Thanks @jpye-finch for your feedback. I have tried to cover up all the variants of the ListRow component.

-> Avatar can be passed in value props. Name with avatar (Like the team page) can be passed in the nameWithAvatar props.
-> Primary button can be passed through action and actionType props, another circle button can be passed through the actionsMenu props.

-> Tag (status) can be passed through the optional status props and circle button through the actionsMenu props.
->HideOnDesktop and HideOnMobile are used for adjusting some styles in desktop and mobile view.
-> MenuContainer and StatusContainer are used for styles of status and actionsMenu props.

Please let me know if you have more queries about these.

File: packages/client/src/views/Home/RecordAudit.tsx (hunk 1 above)
Comment (soumita-dsi) [reply packages/client/src/views/Home/RecordAudit.tsx:247]: @atiqzaman-dsi vai, fixed them by merging the branch with develop.

**Pull request merged**